redash/query_runner/qubole.py

```diff
@@ -0,0 +1,130 @@
+from __future__ import absolute_import
+import os
+import json
+import time
+import datetime
+import httplib2
+
+from redash.query_runner import BaseQueryRunner, register
+from redash.query_runner import TYPE_STRING, TYPE_DATE, TYPE_DATETIME, TYPE_INTEGER, TYPE_FLOAT, TYPE_BOOLEAN
+from redash.utils import json_dumps
+
+
+class Qubole(BaseQueryRunner):
+
+    @classmethod
+    def configuration_schema(cls):
+        return {
+            "type": "object",
+            "properties": {
+                "endpoint": {
+                    "type": "string",
+                    "title": "API Endpoint",
+                    "default": "https://api.qubole.com"
+                },
+                "token": {
+                    "type": "string",
+                    "title": "Auth Token"
+                },
+                "cluster": {
+                    "type": "string",
+                    "title": "Cluster Label",
+                    "default": "default"
+                }
+            },
+            "order": ["endpoint", "token", "cluster"],
+            "required": ["endpoint", "token", "cluster"],
+            "secret": ["token"]
+        }
+
+    @classmethod
+    def enabled(cls):
+        return True
+
+    @classmethod
+    def annotate_query(cls):
+        return False
+
+    def test_connection(self):
+        headers = {"Content-type": "application/json", "Accept": "application/json",
+                   "X-AUTH-TOKEN": self.configuration['token']}
+        h = httplib2.Http()
+        resp, content = h.request("%s/api/latest/users" % self.configuration['endpoint'],
+                                  "HEAD", headers=headers)
+        resp.status == 200
+
+    def run_query(self, query, user):
+        try:
+            import qds_sdk
+            from qds_sdk.qubole import Qubole as qbol
+            from qds_sdk.commands import Command, HiveCommand
+        except ImportError as e:
+            raise(e)
+
+        qbol.configure(api_token=self.configuration['token'],
+                       api_url='%s/api' % self.configuration['endpoint'])
+
+        hc = HiveCommand.create(query=query, label=self.configuration['cluster'])
+        print("Qubole command created with Id: %s and Status: %s", hc.id, hc.status)
```

Comment:
Please change those into `logging.debug` calls instead of `print` statements.

Reply:
done.. but used `logging.info`..